src/sentry/static/sentry/app/views/releasesV2/list/releaseCard.tsx

```diff
@@ -7,93 +7,95 @@ import Count from 'app/components/count';
 import Version from 'app/components/version';
 import {Panel, PanelBody, PanelItem} from 'app/components/panels';
 import ReleaseStats from 'app/components/releaseStats';
-import {ProjectRelease, ReleaseProject} from 'app/types';
+import {Release} from 'app/types';
 import TimeSince from 'app/components/timeSince';
 import {t, tn} from 'app/locale';
 import {AvatarListWrapper} from 'app/components/avatar/avatarList';
-import ProjectBadge from 'app/components/idBadge/projectBadge.jsx';
+import TextOverflow from 'app/components/textOverflow';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
+import Tag from 'app/views/settings/components/tag';
 
 import ReleaseHealth from './releaseHealth';
+import NotAvailable from './notAvailable';
 
 type Props = {
-  release: ProjectRelease;
-  project: ReleaseProject;
+  release: Release;
+  orgSlug: string;
   location: Location;
   reloading: boolean;
 };
 
-const ReleaseCard = ({release, project, location, reloading}: Props) => (
-  <StyledPanel reloading={reloading ? 1 : 0}>
-    <PanelBody>
-      <StyledPanelItem>
-        <HeaderLayout>
-          <VersionColumn>
-            <ColumnTitle>{t('Version')}</ColumnTitle>
-          </VersionColumn>
-          <ProjectsColumn>
-            <ColumnTitle>{t('Project name')}</ColumnTitle>
-          </ProjectsColumn>
-          <CommitsColumn>
-            {release.commitCount > 0 && (
+const ReleaseCard = ({release, orgSlug, location, reloading}: Props) => {
+  const {version, commitCount, lastDeploy, authors, dateCreated} = release;
+  return (
+    <StyledPanel reloading={reloading ? 1 : 0}>
```

Comment:
Panel is passing down all props to div element (even camel-cased ones) - do you have any recommendation on how to approach this?

Reply:
@matejminar Yeah, we need to remove reflexbox from Panel, but in order to do so, because Panel spreads `props` into `Flex`, we have to make sure that no Panels use the `Flex` props